### PR TITLE
Wiki Reorg: PJS Ledger guides

### PR DESCRIPTION
**Pull request by filippoweb3**
See https://github.com/w3f/polkadot-wiki/issues/6085

Task "PJS guide on connecting to Asset Hub ledger app needs update"

Rm outdated content, merge and update general content in Ledger Guides page.

### CODE REVIEW

File: docs/learn/learn-guides-ledger.md [hunk 1]
```diff
@@ -14,11 +14,17 @@ import MessageBox from "../../components/MessageBox"; import "../../components/M
 [Polkadot Support Team](https://support.polkadot.network/support/home). For more user-friendly tools
 see the [wallets](./wallets-index), [apps](./apps-index) and [dashboard](./dashboards-index) pages." />
 
+The [Ledger devices](../general/ledger.md) are hardware wallets that keep your private key secured
+on a physical device not directly exposed to your computer or the internet.
+
+The [Polkadot Generic application](../general/ledger.md#polkadot-generic-app) allows you to manage
+your tokens and other tokens on the Asset Hub and any other chain within the Polkadot ecosystem.
```
Comment (profradha) [inline docs/learn/learn-guides-ledger.md]:
```suggestion
The [Polkadot Generic application](../general/ledger.md#polkadot-generic-app) allows you to manage
your DOT/KSM on Polkadot/Kusama networks, tokens on their Asset Hubs and possibly all chains within the Polkadot ecosystem. It is versatile and capable of handling parachains and relay chains without being affected by their runtime upgrades. 
```

File: docs/learn/learn-guides-ledger.md [hunk 2]
```diff
@@ -50,15 +56,32 @@ used in Ledger Live, as it only scans through the parent accounts. So, remember
...
 
-If you are adding your Ledger Nano for the first time, click on the "Unknown device" line and the
-"Connect" button will become available.
+:::danger Verifying Extrinsics
+
+Visit the
+[**dedicated support page**](https://support.polkadot.network/support/solutions/articles/65000179161-how-can-i-verify-what-extrinsic-i-m-signing-#Verify-an-extrinsic-using-Ledger)
+and see [**this video tutorial**](https://youtu.be/bxMs-9fBtFk?t=360) tutorial to learn how to
+verify extrinsics before signing them. The video will also mention potential attacks that can happen
+to you while signing transactions.
```
Comment (profradha) [inline docs/learn/learn-guides-ledger.md]:
```suggestion
while signing transactions.
```

File: docs/learn/learn-guides-ledger.md [hunk 3]
```diff
@@ -71,45 +94,19 @@ more information about signing transactions using your ledger.
...
 
 ## Receiving a Transfer
 
 To receive a transfer on the accounts stored on your Ledger device, you must provide the sender
-(i.e. the payer) with your address. To do so, follow the instructions on
+(i.e., the payer) with your address. To do so, follow the instructions on
 [this support page](https://support.polkadot.network/support/solutions/articles/65000181866-how-to-receive-dot-to-my-account-on-polkadot-js-ui).
 
-:::warning
+:::caution Before giving anyone your address
```
Comment (profradha) [inline docs/learn/learn-guides-ledger.md]:
```suggestion
:::caution Sharing your account address
```

File: docs/learn/learn-guides-ledger.md [hunk 4]
```diff
@@ -118,21 +115,24 @@ vigilant around copy-paste operations makes sense.
 
 :::
 
-## Staking
+The easiest way to get your address is to click on the account name. This will open a sidebar
+showing the information about your address and other information, such as on-chain identity. Another
```
Comment (profradha) [inline docs/learn/learn-guides-ledger.md]:
```suggestion
showing your address and other information, such as on-chain identity. Another
```

File: docs/learn/learn-guides-ledger.md [hunk 5]
```diff
@@ -118,21 +115,24 @@ vigilant around copy-paste operations makes sense.
...
-the learn how to remove democracy locks after the end of a Governance referendum.
+:::
+
+## Staking
 
-**Please be advised**: Despite the Polkadot ledger application being compatible with both the Ledger
-Nano S and the Ledger Nano X, some extrinsics are not supported by the light version. The following
-[repository by Zondax](https://github.com/Zondax/ledger-polkadot) lists the currently supported
-extrinsics on the XL version of the Ledger app.
+For staking using Ledger devices, see
```
Comment (profradha) [inline docs/learn/learn-guides-ledger.md]:
```suggestion
For staking using Ledger devices, follow the instructions on
```

**Review by DrW3RK — APPROVED** [review]
Added some optional suggestions.

**Pull request merged**